docs/40-troubleshooting.md

```diff
@@ -0,0 +1,18 @@
+# Getting logs without Azure Portal
+Logs on a cluster node are not persisted through Azure Storage. `getClusterLogs` function will fetch the boot up script that users use in [customize script](./30-customize-cluster.md) and [package installation](./20-package-management.md).
+
+```R
+# This call will fetch the stderr.txt file and save it as a text file named "pool-errors.txt"
+getClusterLogs(cluster, "tvm-1170471534_2-20170829t072146z", type = "stderr", localPath = "abc.txt")
```

Comment:
Didn't we get rid of getFooLogs in favor of getFooFiles? Same comment for docs below.